File: Programming Basics/For Loop - Exercises/06.Oscars.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        string actorName = Console.ReadLine();
        double points = double.Parse(Console.ReadLine());
        int graders = int.Parse(Console.ReadLine());

        for (int i = 0; i < graders; i++)
        {
            string graderName = Console.ReadLine();
            double graderPoints = double.Parse(Console.ReadLine());

            if (points < 1250.5)
            {
                points += (graderPoints * graderName.Length) / 2;
            }
        }
        if (points < 1250.50)
        {
            Console.WriteLine($"Sorry, {actorName} you need {1250.50 - points:f1} more!");
        }
        else if (points >= 1250.50)
        {
            Console.WriteLine($"Congratulations, {actorName} got a nominee for leading role with {points:f1}!");
        }
    }
}

File: Programming Basics - Python/For Loop - Exercises/06.Oscars.py
actor_name = input()
points = float(input())
n = int(input())

for i in range(0, n):
    name = input()
    given_points = float(input())

    points += (len(name) * given_points) / 2

    if points >= 1250.5:
        print(f"Congratulations, {actor_name} got a nominee for leading role with {points:.1f}!")
        exit()

print(f"Sorry, {actor_name} you need {abs(points - 1250.5):.1f} more!")